.travis.yml

```diff
@@ -1,29 +1,28 @@
 sudo: false
-
 language: java
 
 jdk:
-- oraclejdk8
-
-addons:
```

Comment:
I'd keep this here, because it installs newest JDK available (if I understand it correctly).